playbook/lib/generators/kit/templates/kit_example_swift.erb.tt

```diff
@@ -0,0 +1,7 @@
+![badge-colors](https://playbook.powerapp.cloud/packs/media/images/pb-logo-7c6d08a9717b9aeec0c40f3e68eb80fd.svg)
```

Comment:
our swift kits don't have code examples, they have png files. This is just the playbook logo